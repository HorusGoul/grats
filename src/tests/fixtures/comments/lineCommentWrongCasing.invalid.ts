// @GQLtYPE
export default class Composer {
  // @gqlfield
  url(): string {
    return `/composer/`;
  }
}
