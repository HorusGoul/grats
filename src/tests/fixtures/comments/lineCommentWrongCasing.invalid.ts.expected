-----------------
INPUT
----------------- 
// @GQLtYPE
export default class Composer {
  // @gqlfield
  url(): string {
    return `/composer/`;
  }
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/comments/lineCommentWrongCasing.invalid.ts:1:4 - error: `@GQLtYPE` is not a valid Grats tag. Valid tags are: `@gqlType`, `@gqlField`, `@gqlScalar`, `@gqlInterface`, `@gqlEnum`, `@gqlUnion`, `@gqlInput`.

1 // @GQLtYPE
     ~~~~~~~~
src/tests/fixtures/comments/lineCommentWrongCasing.invalid.ts:1:4 - error: Unexpected Grats tag in line comment. Grats only supports GraphQL tags in JSDoc-style block comments. e.g. `/** @gqlType */`

1 // @GQLtYPE
     ~~~~~~~~
src/tests/fixtures/comments/lineCommentWrongCasing.invalid.ts:3:6 - error: `@gqlfield` is not a valid Grats tag. Valid tags are: `@gqlType`, `@gqlField`, `@gqlScalar`, `@gqlInterface`, `@gqlEnum`, `@gqlUnion`, `@gqlInput`.

3   // @gqlfield
       ~~~~~~~~~
src/tests/fixtures/comments/lineCommentWrongCasing.invalid.ts:3:6 - error: Unexpected Grats tag in line comment. Grats only supports GraphQL tags in JSDoc-style block comments. e.g. `/** @gqlType */`

3   // @gqlfield
       ~~~~~~~~~
